contracts/HybridCustody.cdc

```diff
@@ -598,6 +628,10 @@ pub contract HybridCustody {
             self.parents[parentAddress] = false
         }
 
+        pub fun check(): Bool {
```

Comment:
Nitpick: Prefer to avoid using method names found in the language API as it can get confusing - e.g. `Capability.check()` and `ChildAccount.check()` here. Not that big of a deal, but could get confusing in the case you have: `child.check()` where `child` could be `Capability` or `&ChildAccount`

Reply:
Good callout, makes sense!